src/components/layout/MainLayout/MainLayout.js

```diff
@@ -7,19 +7,30 @@ import Brands from '../../features/Brands/BrandsContainer';
 import StickyBarContainer from '../StickyBar/StickyBarContainer';
 import LatestBlog from '../../features/LatestBlog/LatestBlogContainer';
 
-const MainLayout = ({ children }) => (
-  <div>
-    <Header />
-    <StickyBarContainer />
-    {children}
-    <LatestBlog />
-    <Brands />
-    <Footer />
-  </div>
-);
+const MainLayout = ({ children, changeMode }) => {
+  useEffect(() => {
+    window.addEventListener('resize', changeMode);
+
+    return () => {
+      window.removeEventListener('resize', changeMode);
+    };
+  });
```

Comment:
Poszukaj w Google jak zastąpić `componentDidMount` za pomocą `useEffect` bo właśnie to musisz tutaj zrobić, przy okazji w tedy możesz poczytać o tym jak działa `useEffect` ;)